app/authConfig.js

```diff
@@ -5,8 +5,11 @@
  */
 const msalConfig = {
     auth: {
+        // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
         clientId: "Enter_the_Application_Id_Here",
-        authority: "Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here",
+        // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+        authority: "Enter_the_Cloud_Instance_Id_Here/Enter_the_Tenant_Info_Here",
```

Comment:
```suggestion
        authority: "Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here",
```

@henrymbuguakiarie unfortunatelly we need to omit the slash here, since the Azure portal quickstart template replaces this string automatically with a cloud instance string ending with a trailing slash